### PR TITLE
WebsiteTabs fixes

### PR DESCRIPTION
**Pull request by lovemecomputer**
Closes https://github.com/carbon-design-system/carbon-website/issues/943

temporary solution for https://github.com/carbon-design-system/carbon-website/issues/942 - need to eventually circle back and make a longer-term solution which doesn't require an author to add `href="javascript:void 0"` to every `<tab>` component. thanks @asudoh !

### CODE REVIEW

File: src/components/WebsiteTabs/WebsiteTabs.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ export default class WebsiteTabs extends React.Component {
   render() {
     this.updateTabChildren();
 
+    console.log(this.tabChildren[0].props);
```
Comment (dakahn) [inline src/components/WebsiteTabs/WebsiteTabs.js]: left a log in

**Comment by tw15egan** [comment]
Looks like this can be turned off from the package: https://www.npmjs.com/package/smooth-scroll

Just need to set `topOnEmptyHash: false` 

screen shot 2019-02-08 at 4 26 27 pm

File: src/components/WebsiteTabs/WebsiteTabs.js (hunk 1 above)
Comment (lovemecomputer) [reply src/components/WebsiteTabs/WebsiteTabs.js]: @dakahn thanks!

**Comment by lovemecomputer** [comment]
thanks @tw15egan ! looks like all is good with this PR now~

**Pull request merged**